components/Step2/Step2.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {Image, Text, View} from 'react-native';
+import RadioButton from '../RadioButton/RadioButton';
+import constants from '../../utils/constants';
+import {updateUserData} from '../../redux/reducers/WizardData';
+import {useDispatch, useSelector} from 'react-redux';
+
+import globalStyle from '../../assets/styles/globalStyle';
+
+export const Step2 = () => {
+  const dispatch = useDispatch();
+  const wizardData = useSelector(
+    (state: {wizard: {level: string}}) => state.wizard,
+  );
+
+  const data = [
```

Comment:
`data` `value` `v` `i` `e` are all bad candidates for names. Please add some context to the name. 

`data` gives me no information whatsoever, and reading the object fields gives me no information again, because I don't know where it's being used. However, changing it to `radioButtons` would be 100% better and give me all of the necessary info immediately (what it is and where it's used)